src/components/__tests__/action-button.spec.js

```diff
@@ -16,152 +16,155 @@ describe('action button', () => {
-      defaults.layout.navigation = { action: 'firstSheet', sheet: 'mySheet' };
+      defaults.layout.actions = [{ actionType: "applyBookmark" }, { actionType: "clearAll" }];
+      defaults.layout.navigation = { action: "firstSheet", sheet: "mySheet" };
       defaults.app.clearAll = jest.fn();
       defaults.senseNavigation = {
         goToSheet: jest.fn(),
         getCurrentStoryId: () => false,
       };
     });
-    it('should render action button', () => {
+    it("should render action button", () => {
       renderButton(defaults);
       expect(defaults.element.firstElementChild).toBeInstanceOf(Object);
-      expect(button.setAttribute).toHaveBeenCalled;
+      expect(button.setAttribute).toHaveBeenCalled();
     });
 
-    it('should disable action button on click and enable again', async () => {
+    it("should disable action button on click and enable again", async () => {
       renderButton(defaults);
       await defaults.element.firstElementChild.onclick();
-      expect(button.setAttribute).toHaveBeenCalledWith('disabled', true);
-      expect(button.removeAttribute).toHaveBeenCalledWith('disabled');
-      expect(defaults.senseNavigation.goToSheet).toHaveBeenCalled;
+      expect(button.setAttribute).toHaveBeenCalledWith("disabled", true);
+      expect(button.removeAttribute).toHaveBeenCalledWith("disabled");
+      expect(defaults.senseNavigation.goToSheet).toHaveBeenCalled();
     });
 
-    it("should not act on click when 'active' constraint is enabled ", async () => {
+    it("should not act on click when 'active' constraint is enabled", async () => {
       defaults.constraints = { active: true };
       renderButton(defaults);
       await defaults.element.firstElementChild.onclick();
-      expect(button.setAttribute).not.toHaveBeenCalledWith('disabled', true);
+      expect(button.setAttribute).not.toHaveBeenCalledWith("disabled", true);
     });
 
-    it('should not act on click when button is disabled by condition', async () => {
+    it("should not act on click when button is disabled by condition", async () => {
       defaults.layout.useEnabledCondition = true;
       defaults.layout.enabledCondition = 0;
       renderButton(defaults);
       await defaults.element.firstElementChild.onclick();
-      expect(button.setAttribute).not.toHaveBeenCalledWith('disabled', true);
+      expect(button.setAttribute).not.toHaveBeenCalledWith("disabled", true);
     });
 
-    it('should run without navigation', async () => {
+    it("should run without navigation", async () => {
       defaults.layout.navigation = {};
       renderButton(defaults);
       await defaults.element.firstElementChild.onclick();
-      expect(button.removeAttribute).toHaveBeenCalledWith('disabled');
-      expect(defaults.senseNavigation.goToSheet).toNotHaveBeenCalled;
+      expect(button.removeAttribute).toHaveBeenCalledWith("disabled");
+      expect(defaults.senseNavigation.goToSheet).toHaveBeenCalledTimes(0);
```

Comment:
Actually think this was not asserting anything at all. This should be the correct pattern